### PR TITLE
implement DetailsTile in PeopleDetails component

### PR DESCRIPTION
**Pull request by patrycja-gomza**
## 🚀 Pull Request: Adding DetailsTile to PeopleDetails

Hey team! 👋 I've implemented a DetailsTile, created by @bedicooper, into the PeopleDetails component. Now we're showcasing the actor's poster, name, biography, date of birth and place of birth. Everything's in a nice and cohesive layout for an even better look! 😊

If everything looks good, give it a thumbs up 🟢
Thanks for your time and attention! 💙

### CODE REVIEW

**Review by widmo200 — COMMENTED** [review]
it's ok but my suggestion pls change width in the article . In figma project this article dont have 100% width  :P

File: src/common/Tile/index.js [hunk 1]
```diff
@@ -157,7 +157,7 @@ export const DetailsTile = (
 						<TileDataContent>
 							<TileDataTitle>
 								{
-									movieTile ? `Production: ` : `Date of birth: `
+									movieTile ? `Production: ` : `Place of birth: `
 								}
```
Comment (bedicooper) [inline src/common/Tile/index.js:161]: Thanks for correcting that mistake, actually this line should be:
```
movieTile ? `Release date: ` : `Place of birth: `
```

File: src/features/people/PeopleDetails/index.js [hunk 2]
```diff
@@ -24,12 +24,12 @@ const PersonDetails = () => {
...
-                        <PersonDetailsTile
-                            poster={details.profile_path}
+                        <DetailsTile
+                            posterPath={details.profile_path}
                             title={details.name}
-                            overview={details.biography}
-                            dateOfBirth={details.birthday}
-                            placeOfBirth={details.place_of_birth}
+                            description={details.biography}
+                            firstData={details.birthday}
```
Comment (bedicooper) [inline src/features/people/PeopleDetails/index.js:31]: remember to format the date, I suggest running API response through a function to modify the raw data

**Review by bedicooper — APPROVED** [review]
Thanks for that PR and catching my mistake in Tile/index.js
Remember to wrap Your code in <Main> component as it centers and restricts content width 😄

**Comment by patrycja-gomza** [comment]
> it's ok but my suggestion pls change width in the article . In figma project this article dont have 100% width :P

Thanks, @widmo200 for your feedback! 🙌 I appreciate your suggestion. I'll make sure to address the width of the article in the PeopleDetails component. In my upcoming PR, I plan to wrap the code in the <Main> component, as it not only centers the content but also restricts the width, aligning with the design in Figma. 😄

**Comment by patrycja-gomza** [comment]
> Thanks for that PR and catching my mistake in Tile/index.js Remember to wrap Your code in `<Main>` component as it centers and restricts content width 😄

Hi @bedicooper  👋,
You're welcome! 🙌 I'm glad I could assist, and I appreciate your feedback. 😊
Absolutely, I'll make sure to wrap the code in the <Main> component in the upcoming changes. Thanks for the reminder! 😄

File: src/features/people/PeopleDetails/index.js (hunk 2 above)
Comment (patrycja-gomza) [reply src/features/people/PeopleDetails/index.js:31]: Thanks for the heads up! I noticed your suggestion about date formatting and agree that it's an important improvement. I will address this in future improvements and apply the solution with a function analogous to your solution.
Your input is valuable and I'm committed to work out the details to get a refined result 😊🚀

File: src/common/Tile/index.js (hunk 1 above)
Comment (patrycja-gomza) [reply src/common/Tile/index.js:161]: You're welcome! 🙌 I appreciate your feedback, and I've duly noted the correction. I'll make sure to update the line to:
`movieTile ? `Release date: ` : `Place of birth: ``
Thanks for catching that! 😊🚀

**Pull request merged**